feanil/upgrade bleach

feanil opened the pull request:
- fix: Prepare for the bleach 6.0.0 upgrad.
- chore: Compile requirements to update `bleach`

Commit:
fix: Prepare for the bleach 6.0.0 upgrad.

Changelog: https://bleach.readthedocs.io/en/latest/changes.html#version-6-0-0-january-23rd-2023

The major change is that the tags and protocols attributes and related
constants are expected to be sets rather than lists.

Commit:
chore: Compile requirements to update `bleach`

Run `make compile-requirements COMPILE_OPTS="-P 'bleach'"` to update
bleach to the latest version.

Commit:
chore: Update openedx-django-wiki

Run `make compile-requirements COMPILE_OPTS="-P 'openedx-django-wiki'"`
to update `openedx-django-wiki`

aht007 commented:
@feanil make upgrade is failing in edx-platform due to newer version of `djano-wiki`. I am pinning it in this [PR](https://github.com/openedx/edx-platform/pull/31681) to resolve the upgrade job issue for now. You can remove it as a part of this PR.

Commit:
chore: Update ora2 and lti-consumer-xblock.

Run `make compile-requirements COMPILE_OPTS="-P 'ora2' -P 'lti-consumer-xblock'"`
to update two more dependencies that use bleach.

Commit:
chore: Upgrade xblock-drag-and-drop-v2

Run `make compile-requirements COMPILE_OPTS="-P 'xblock-drag-and-drop-v2'"`
